certbot/CHANGELOG.md

```diff
@@ -10,7 +10,7 @@ Certbot adheres to [Semantic Versioning](https://semver.org/).
 
 ### Changed
 
-*
+* Added a new signing key, `F2871B4152AE13C49519111F447BF683AA3B26C3`
```

Comment:
I'd like to make this a little more verbose to clarify what this signing key is for.
```suggestion
* The PGP key `F2871B4152AE13C49519111F447BF683AA3B26C3` was added as an
  additional trusted key to sign our PyPI packages
```